frame/vesting/src/lib.rs

```diff
@@ -567,20 +567,29 @@ impl<T: Config> Pallet<T> {
 		(BalanceOf<T>, BoundedBTreeMap<T::VestingScheduleId, BalanceOf<T>, T::MaxVestingSchedules>),
 		DispatchError,
 	> {
-		let mut claims_per_schedule: BoundedBTreeMap<
-			T::VestingScheduleId,
-			BalanceOf<T>,
-			T::MaxVestingSchedules,
-		> = BoundedBTreeMap::new();
-
 		<VestingSchedules<T>>::try_mutate_exists(who, asset, |maybe_schedules| {
-			let mut total_balance_to_claim: BalanceOf<T> = Zero::zero();
 			match maybe_schedules {
```

Comment:
this `match` could be `let schedules = maybe_schedules.ok_or(Error::<T>::VestingScheduleNotFound)?`